portal/src/texts/profile/a11y.md

```diff
@@ -1,8 +1,10 @@
 ## Tilgjengelighet
 
-Universell utforming handler om at vi skal lage løsningene våre med tanke på at brukere kan ha ulik funksjonsevne, midlertidig eller varig. Når vi tenker på å utforme tilgjengelige løsninger, når vi alle målgruppene med én og samme løsning.
+Universell utforming handler om at vi lager løsninger med tanke på at brukere kan ha ulik funksjonsevne. Den kan være  midlertidig eller varig. Eksempler kan være synshemming, nedsatt finmotorikk eller vansker med å skille farger. Når vi planlegger hvordan vi skal utforme løsninger som er tilgjengelig for alle, får vi én løsning som når alle målgrupper.
+
 En løsning er universelt utformet når den
-gir alle brukere samme muligheter, uansett funksjonsevne
-tilpasser seg brukeren uansett hvor i løsningen de befinner seg
 
-I løsninger som er rettet mot kunder, må Fremtinds produkter følge WCAG-standarden. Den spesifiserer hvordan ulike deler av løsningen skal være best mulig tilgjengelig for alle. Reglene gjelder ikke på samme måte for interne saksbehandlingsløsninger, men ved å følge gode prinsipper for design og tilgjengelighet, oppnår vi de beste løsningene uansett hvem som skal bruke dem.
+- gir alle brukere samme muligheter til å bruke den, uansett funksjonsevne
+- tilpasser seg brukeren, uansett hvor i løsningen de befinner seg
+
+I løsninger som er rettet mot kunder, må Fremtinds produkter følge WCAG-standarden. Den spesifiserer hvordan ulike deler av løsningen skal være best mulig tilgjengelig for alle. Reglene gjelder ikke på samme måte for interne saksbehandlingsløsninger. Målet vårte er likevel at vi skal følge gode prinsipper for design og tilgjengelighet, slik at vi får de beste løsningene, uansett hvem som skal bruke dem.
```

Comment:
If we keep text as is, `Målet vårte*` should be `Målet vårt`

Reply:
Text has been changed, so vårte is "bårte" :-)